timemachine/potentials/chiral_restraints.py

```diff
@@ -110,7 +110,7 @@ def chiral_atom_restraint(conf, params, box, lamb, idxs):
     return jnp.sum(U_chiral_atom_batch_all(conf, idxs, params)) if len(idxs) else 0.0
 
 
-def chiral_bond_restraint(conf, params, box, lamb, idxs, signs):
+def chiral_bond_restraint(conf, params, box, idxs, signs):
```

Comment:
nit: delete lamb reference in docstrings, L107, 119

Reply:
Fixed in 370861d